docs/advanced-usage.rst

```diff
@@ -0,0 +1,12 @@
+Advanced usage
+==============
+
+ParallelRunner
+--------------
+
+A parallel runner, leveraging multi-core systems to run multiple simulations in
+parallel, can be used instead of the standard SimulationRunner. Please note that
+support for this feature is still experimental. To perform simulations in
```

Comment:
"this feature is still experimental".

Will you plan commit a stable version of parallel runner in the end of GSoC?

Reply:
Yes, absolutely! I haven't had the time to test this thoroughly yet, however so far it has worked without any problems. I will better document it (and maybe find a more natural way to select it).

Related to this, I was thinking about implementing integrations with cluster software, since the next two weeks in my plan were allocated to developing the parallel runner solution (which already works). What do you think? I have access to a Sun Grid Engine computing facility at my university, and I've seen that there already exist a library that does the heavy lifting for the integration, so I think that could be a nice target for an initial implementation. After this example, we could open to developers integrating custom runners for their cluster architecture!